electron/settings.html

```diff
@@ -615,4 +647,5 @@ <h2>Our Scout-Captain Level Supporters</h2>
 <script src="vendor/toggle/bootstrap-toggle.min.js"></script>
 <link href="vendor/bootstrap-pin-input/bootstrap-pincode-input.css" rel="stylesheet">
 <script src="vendor/bootstrap-pin-input/bootstrap-pincode-input.js"></script>
-</html>
\ No newline at end of file
+<script type="text/javascript" src="misc_helpers.js"></script>
```

Comment:
Didn't think about that. Will fix.

Reply:
Actually, I just removed the file, since I ended up not needing to share the function. 

Sidenote: I'm thinking of doing a wash-down of the code, unifying style, variable names, etc., moving stuff to includes, that kind of thing. Though I admit I'm not in a rush to do it.

Reply:
Same. I'll be the first to admit: splitting between python snake_case and js camelCase, the style in this project is an utter mess. I've been putting it off until we have some semblance of regression tests, since it would be super stupid to accidentally break stuff over style.